build_scripts/build_usd.py

```diff
@@ -1161,13 +1341,160 @@ def InstallPNG(context, force, buildArgs):
     OPENEXR_URL = "https://github.com/AcademySoftwareFoundation/openexr/archive/refs/tags/v2.5.2.zip"
 else:
     OPENEXR_URL = "https://github.com/AcademySoftwareFoundation/openexr/archive/refs/tags/v2.4.3.zip"
+    
+def updateOpenEXRIOS(context, srcDir):
+    # IlmBase
```

Comment:
let's fix OpenEXR upstream. It's okay to depend on a later version of OpenEXR specifically for iOS. I'm able to make commits in the OpenEXR repo, so I can stand by to help get these fixes landed as quickly as possible.

Reply:
Agree to that!

Reply:
Hm, OpenColorIO will not respect -DUSE_OpenEXR=OFF

Reply:
So for the time being I believe we won't include openimageio
https://github.com/OpenImageIO/oiio/blob/master/INSTALL.md
from my side, it's a reasonable step to only use jpgs and minimal textures to begin with and openimagiio can then be added later

Reply:
That sounds like a good intermediate step in order to get things up and running.